### DISCUSSION TITLE
`remix vite:build` results React to run in development mode

### DISCUSSION THREAD

**Discussion by shirshendubhowmick** [remix-run/remix#10280] [Q&A]
I have a Remix project setup with Vite, I migrated from remix compiler to vite using the official migration guide doc.

However when I build the app and serve either locally using Remix server or on vercel, I see React dev tools saying its running on development mode (the red React dev tool icon).

When I run `remix vite:build` I can even see longs which says 

```
vite v5.4.11 building for production...
```
I am on
Remix `v2.15.0`
Vite: `v5.4.11`
Not really sure whats going wrong here. Any help would be appericiated.

**Comment by keul** [comment]
Noticed the same today. I would expect that `remive vite:build` ends into the MODE='production' envvar
